lib/init.js

```diff
@@ -0,0 +1,82 @@
+'use babel';
+
+import { parse, exec } from 'atom-linter';
+import { dirname, extname } from 'path';
+
+const isWindows = (/^win/).test(process.platform);
+const pattern = '.+:(?<line>\\d+):(?<col>\\d+)-(?<colEnd>\\d+):' +
+  ' \\((?<type>[EW])\\d+\\) (?<message>.*)';
+
+const checkedAppend = function checkedAppend(parameters, opt, args) {
+  if (args.length > 0) {
+    parameters.push(opt);
+    return parameters.concat(args);
+  }
+
+  return parameters;
+};
+
+const makeParameters = function makeParameters(globals, ignore, file) {
```

Comment:
I placed this in the returned object, this allows me to use `this.globals`, `this.ignore` and so on without passing them as arguments... is that fine?